### PR TITLE
Fix: Broken `XML` due to Title and Urls in `RSS`

### PR DESCRIPTION
**Pull request by ItzNotABug**
## What does this PR do?

An `&` and a ` ` (space) broke the xml format. This should fix the issue.

## Test Plan

Manual check via `https://validator.w3.org/feed`.

## Related PRs and Issues

* #974.

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes.

### CODE REVIEW

File: src/routes/blog/rss.xml/+server.ts [hunk 1]
```diff
@@ -3,23 +3,40 @@ import { posts } from '../content';
...
     <description>Appwrite is an open-source platform for building applications at any scale, using your preferred programming languages and tools.</description>
     ${posts.map((post) => `<item>
-        <title>${post.title}</title>
+        <title>${encodeTitle(post.title)}</title>
         <pubDate>${post.date.toUTCString()}</pubDate>
-        <link>https://appwrite.io${post.href}</link>
-        <guid>https://appwrite.io${post.href}</guid>
+        <link>${encodeUrl(post.href)}</link>
+        <guid>${encodeUrl(post.href)}</guid>
         <description>${post.description}</description>
```
Comment (TorstenDittmann) [inline src/routes/blog/rss.xml/+server.ts:31]: I tihnk descrition needs to be encoded too - I am not sure about the urls though

Comment (ItzNotABug) [reply src/routes/blog/rss.xml/+server.ts:31]: I could add encoding to description, kinda makes sense.
One of the urls had a space and that would really break the xml, so I added encoding to urls too.

**Comment by ItzNotABug** [comment]
@TorstenDittmann should we keep description if `null`?

**Comment by TorstenDittmann** [comment]
> @TorstenDittmann should we keep `description` if `null`?

There should always be a description 👍🏻 

can we check and use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURI for the urls?

**Comment by ItzNotABug** [comment]
> > @TorstenDittmann should we keep `description` if `null`?
> 
> There should always be a description 👍🏻
> 
> can we check and use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURI for the urls?

I am actually using `encodeURI`, just in a [function](https://github.com/appwrite/website/blob/07f5c77083c3fc35086e723f110398a533eedea9/src/routes/blog/rss.xml/%2Bserver.ts#L16C1-L18C2). Or am I missing something here 🤔?

**Review by TorstenDittmann — APPROVED** [review]

**Pull request merged**